docs/pages/database-access/guides/oracle-self-hosted.mdx

```diff
@@ -0,0 +1,266 @@
+  Service.
+
+</Admonition>
+
+Start the database service:
+
+```code
+$ teleport start --config=/path/to/teleport-db.yaml --token=/tmp/token
+```
+
+</TabItem>
+</Tabs>
+
+<Admonition type="tip">
+  You can start the Database Service using a configuration file instead of CLI flags.
+  See the [YAML reference](../reference/configuration.mdx) for details.
+</Admonition>
+
+## Step 2/5. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+
+## Step 3/5. Create a certificate/key pair and create Teleport Oracle Server Wallet
+
+(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)
+
+Follow the instructions below to generate TLS credentials for your database.
+
+    ```code
+    # Export Teleport's certificate authority and a generate certificate/key pair
+    # for host db.example.com with a 1-year validity period.
+    $ tctl auth sign --format=oracle --host=db.example.com --out=server --ttl=2190h
+    ```
+
+    (!docs/pages/includes/database-access/ttl-note.mdx!)
+
+If [Orapki](https://docs.oracle.com/en/database/oracle/oracle-database/21/dbseg/using-the-orapki-utility-to-manage-pki-elements.html#GUID-0F77F9B8-D425-4A0E-86FC-359D72DAD34F) Oracle tool was found in your local environment the `tctl auth sign --format=oracle --hostdb.example.com --out=server --ttl=2190h` command will produce Oracle Wallet and
+instructions how to configure Oracle TCPS listener with Teleport Oracle Wallet. Otherwise the `tctl auth sign --format=oracle`  will produce `p12` certificate and instructions on how to create an Oracle Wallet on Oracle Server instance.
+
+## Step 4/5. Configure Oracle Server
+
+In order to enable Teleport Oracle integration you will need to configure the TCPS Oracle listener and use the Teleport Oracle Wallet created in the previous step
+
+Align your `listener.ora` Oracle configuration file and add the following entries:
+```
+LISTENER =
+  (DESCRIPTION_LIST =
+    (DESCRIPTION =
+      (ADDRESS = (PROTOCOL = TCPS)(HOST = 0.0.0.0)(PORT = 2484))
+    )
+  )
+
+WALLET_LOCATION = (SOURCE = (METHOD = FILE)(METHOD_DATA = (DIRECTORY = /path/to/oracleWalletDir)))
+SSL_CLIENT_AUTHENTICATION = TRUE
+```
+
+
+
+Additionally, you will need to extend your `sqlnet.ora` Oracle configuration:
+```
+WALLET_LOCATION = (SOURCE = (METHOD = FILE)(METHOD_DATA = (DIRECTORY = /path/to/oracleWalletDir)))
+SSL_CLIENT_AUTHENTICATION = TRUE
+SQLNET.AUTHENTICATION_SERVICES = (TCPS)
+```
+
+You can find learn more about  `sqlnet.ora` and `listener.ora` configuration from the [Parameters for the sqlnet.ora File](https://docs.oracle.com/en/database/oracle/oracle-database/18/netrf/parameters-for-the-sqlnet-ora-file.html#GUID-28040885-6832-4FFC-9258-0EF19FE9A3AC) and [Oracle Net Listener Parameters in the listener.ora File](https://docs.oracle.com/en/database/oracle/oracle-database/18/netrf/Oracle-Net-Listener-parameters-in-listener-ora-file.html#GUID-F9FA0DF5-2FAF-45CA-B6A1-F0166C7BFE54) Oracle documentation.
```

Comment:
I think we should put this in the "Next steps" section since it's not directly relevant to completing the instructions in this guide.

Reply:
I think that in same cases this will be useful. For example a custom have a custom Oracle configuration and need to merge the settings but agree this is not related to any instruction step. 

I have move this to `<Admonition type="tip">`

Reply:
As the `Admonition` exists in the guide right now, I'm not sure why we would include links to this information instead of other technical topics. 

If we want the reader to follow these links in order to do something specific, I think we should be explicit about what that is.